site/docs/v1/tech/apis/identity-providers/links.adoc

```diff
@@ -35,25 +38,25 @@ link:/docs/v1/tech/apis/authentication#api-key-authentication[icon:lock[type=fas
 ==== Request Body
 
 [.api]
-[field]#displayName# [type]#[String]# [optional]#Optional# [since]#Available since 1.28.1#::
-A human readable name for this link. This value should be used to make it easier to identify the user this link represents in the remote identity provider.
+[field]#identityProviderLink.displayName# [type]#[String]# [optional]#Optional# [since]#Available since 1.28.1#::
+A human-readable name for this link when initially created. This value should be used to make it easier to identify the user this link represents in the remote identity provider.
 +
-This value is optional and it will always be set by FusionAuth the next time this link is used to resolve the FusionAuth user during a login event for this IdP.
+After the link is created, and a user first successfully logs in with the IdP they are linked to, this value will typically be set to either an email or username.
 
 
-[field]#identityProviderId# [type]#[UUID]# [required]#Required#::
+[field]#identityProviderLink.identityProviderId# [type]#[UUID]# [required]#Required#::
 The Id of the identity provider. This identity provider must exist.
```

Comment:
```suggestion
The Id of the Identity Provider. This Identity Provider must exist.
```

We're not super consistent about this, but whenever we are referring to a FusionAuth config object, we should capitalize it.